types/barco-eventmaster/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "dom"],
```

```suggestion
        "lib": ["es6"],
```
can you also remove 'console...' from tests? that will remove dependency on DOM lib, where there is no one